doc/Options/z390_options.md

```diff
@@ -7,6 +7,12 @@ The scope value details where the option is used.
 * L - Linker
 * E - Execution
 
+!!! Note "Use of parameters with parentheses in *nix shells"
+    Unix shells like sh and bash interpret parentheses if not quoted. For this reason, if you need to specify a z390 parameter that contains
+    parenthisis, then that parameter should be quoted. For example, the SYSMAC parameter is quoted as it uses parenthesis, whereas the ALIGN parameter is not.
```

Comment:
Two typos on line 12. "parenthesis, then that parameter ... as it uses parenthesis, ..." should be "parentheses, then that parameter ... as it uses parentheses, ..."

Reply:
Resolved by 52df1cf. I have done a full spell check over the code.. Using US spelling for consistency.